core/src/main/scala/org/apache/spark/api/r/RBackendHandler.scala

```diff
@@ -148,6 +151,9 @@ private[r] class RBackendHandler(server: RBackend)
       case e: Exception =>
         logError(s"$methodName on $objId failed", e)
```

Comment:
Yes, there is a limit for ```warning.length``` and default is 1000.
<https://stat.ethz.ch/R-manual/R-devel/library/base/html/options.html>